maintainers-info-report/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+@Library('pipeline-library@pull/348/head') _
+
+pipeline {
+	triggers {
+    // Execute once per hour
+		cron('@hourly')
+	}
+	options {
+		// This pipeline takes 30-40 minutes to execute
+		timeout(time: 1, unit: 'HOURS')
+    lock(resource: 'infra-reports-maintainers-info', inversePrecedence: true)
```

Comment:
Same note as above, not originally locked.

Reply:
It wasn't on the previous job, but having PRs and shorter time between jobs, sonds safer (hence the addon)